docs/source/system_reference_guide/s3-inventory.ipynb

```diff
@@ -0,0 +1,2025 @@
+{
```

Comment:
<p>Probably don't need to iterate through objects if we're just gather the prefixes under file-staging/nasa-map</p><p> </p><pre class="ql-syntax" spellcheck="false">import boto3

s3 = boto3.client('s3')
bucket_name = 'nasa-maap-data-store'
prefix = 'file-staging/nasa-map/'

# List the prefixes under the specified prefix
response = s3.list_objects_v2(Bucket=bucket_name, Prefix=prefix, Delimiter='/')

# Get the common prefixes
common_prefixes = response.get('CommonPrefixes')


# Print the common prefixes
for common_prefix in common_prefixes:
&nbsp; &nbsp; print(common_prefix['Prefix'])
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>